src/Fantomas.Core/CodePrinter.fs

```diff
@@ -1401,9 +1401,12 @@ and genExpr astContext synExpr ctx =
             )
 
         | IndexWithoutDotExpr (identifierExpr, indexExpr) ->
+
```

Comment:
Nitpick, could you remove that first new line please?

Reply:
Seems like the right project for mierenneuker ;-) (my partner says that, hopefully it's light-hearted!)

Done.